content/references/http-websocket-apis/public-api-methods/account-methods/account_info.md

```diff
@@ -63,7 +63,7 @@ The request contains the following parameters:
 | `ledger_hash`  | String               | No        | A 20-byte hex string for the ledger version to use. (See [Specifying Ledgers][]) |
 | `ledger_index` | Number or String     | No        | The [ledger index][] of the ledger to use, or a shortcut string to choose a ledger automatically. (See [Specifying Ledgers][]) |
 | `queue`        | Boolean              | No        | If `true`, return stats about [queued transactions](transaction-queue.html) sent by this account. Can only be used when querying for the data from the current open ledger. Not available from servers in [Reporting Mode][]. |
-| `signer_lists` | Boolean              | No        | If `true`, return any [SignerList objects](signerlist.html) associated with this account. |
+| `signer_lists` | Boolean              | No        | [API v1][]: If `true`, return any [SignerList objects](signerlist.html) associated with this account.<br>[API v2][]: Identical to v1, but also returns an `invalidParams` error if a non-boolean value is used. |
```

Comment:
Avoid HTML in tables for Redocly compatibility; also, I suggest rephrasing to active voice.

```suggestion
| `signer_lists` | Boolean              | No        | [API v1][]: If `true`, return any [SignerList objects](signerlist.html) associated with this account. [API v2][] returns an `invalidParams` error if you provide a non-boolean value. |
```